app/components/Header/Header.css

```
@@ -21,10 +21,15 @@
   justify-content: space-between;
   align-items: center;
   cursor: pointer;
+  transition: background-color var(--linear-medium);
 
   & button {
     padding: 0;
   }
+
+  &:hover {
+    background-color: var(--color-gray-1);
```

I'd like this to match the rest of the webapp.
```suggestion
    background-color: var(--additive-background);
```

---

You're welcome! Fixed! 👍